client/src/components/SelectDropdown/SelectDropdown.css

```diff
@@ -0,0 +1,12 @@
+.selectDropdown {
```

Comment:
Can you make classes all lowercase and separated by dashes? I.e. `select-dropdown`?